taqueria-protocol/Task.ts

```diff
@@ -0,0 +1,68 @@
+import {z} from 'zod'
```

Comment:
Wow I'm loving the sdk refactor @mweichert, code is much more flexible, semantic, and clean!